datafusion/physical-expr/src/partitioning.rs

```diff
@@ -24,8 +24,8 @@ use crate::{physical_exprs_equal, EquivalenceProperties, PhysicalExpr};
 
 /// Output partitioning supported by [`ExecutionPlan`]s.
 ///
-/// When `executed`, `ExecutionPlan`s  produce one or more independent stream of
-/// data batches in parallel, referred to as partitions. The streams are Rust
+/// Calling [`ExecutionPlan::execute`] produce one or more independent streams of
+/// [`RecordBatch`]es in parallel, referred to as partitions. The streams are Rust
 /// `async` [`Stream`]s (a special kind of future). The number of output
```

Comment:
perhaps we can add the link to the `futures-core` crate which backs streams ?

Reply:
I double checked by building the docs locally

This link looks like this:
<img width="1042" alt="Screenshot 2024-08-25 at 6 59 37 AM" src="https://github.com/user-attachments/assets/3d71d743-1fb5-4fb7-bd68-68c084422ca5">


And it goes to this link:
https://docs.rs/futures/latest/futures/stream/trait.Stream.html

Is that what you mean by link to `futures-core`? Or do you mean something else?

Reply:
exactly, my bad, when I navigated from IDE it showed me the crate name as `futures-core` sub crate which is confusing